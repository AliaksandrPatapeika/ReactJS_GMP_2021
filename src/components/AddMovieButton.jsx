import '../less/components/AddMovieButton.less';

import React from 'react';

const AddMovieButton = () => <button type="button" className="AddMovieButton">+ ADD MOVIE</button>;

export default AddMovieButton;
