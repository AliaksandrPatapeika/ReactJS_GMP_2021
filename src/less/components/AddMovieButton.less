@import "../variables";

.AddMovieButton {
    cursor: pointer;
    width: 184px;
    color: @red;
    height: 48px;
    background-color: @gray;
    border-radius: 4px;
    opacity: 0.8;
    font-family: @NetflixSans-Medium;
    font-size: 22px;
    outline: none;
}