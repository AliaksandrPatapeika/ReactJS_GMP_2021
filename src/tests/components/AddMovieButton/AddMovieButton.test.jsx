import React from 'react';
import renderer from 'react-test-renderer';

import AddMovieButton from '../../../components/AddMovieButton';

describe('AddMovieButton', () => {
  test('renders correctly', () => {
    const addMovieButton = renderer
      .create(<AddMovieButton />)
      .toJSON();

    expect(addMovieButton).toMatchSnapshot();
  });
});
