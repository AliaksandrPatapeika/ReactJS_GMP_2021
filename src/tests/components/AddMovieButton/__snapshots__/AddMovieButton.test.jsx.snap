// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMovieButton renders correctly 1`] = `
<button
  className="AddMovieButton"
  type="button"
>
  + ADD MOVIE
</button>
`;
